test.sh

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
 
 # main test runner for heketi
+failures=''
+function run-test()
+{
+	$@ || failures=$( [ -n "$failures" ] && echo "$failures\\n$@" || echo "$@" )
```

Comment:
Shouldn't this be quoted as follows?

```
failures="$( ... || echo "$@" )" 
```

Reply:
I don't think so. And it's been proven to work well in mgmt for a while now.

Reply:
Well I think so. At least when I add shellcheck tests, this will be needed, I guess. Maybe it is never triggered the way we use it, but i made a habit of quoting all such variable assignment/expansions to keep checks happy. Well, we could still add this if some checker fails... ;-)